docs/native/src/issues/invalidcharacters.rst

```diff
@@ -0,0 +1,9 @@
+
+|Office iOS| will fail to load files with invalid characters in the file name, folder names or user ID. 
+=======================================================================================================
+
+If a file name, folder name or user ID contains the following characters, the file will not open properly in |Office iOS|:
```

Comment:
Can you re-word this in terms of the WOPI CheckFileInfo fields that can't contain these characters? Something like:

> Office iOS will fail to load files if there are invalid characters in certain CheckFileInfo or CheckContainerInfo properties. These properties are affected:
>
> ### CheckFileInfo
> * BaseFileName
> * UserId
> * OwnerId
>
> ### CheckContainerInfo
> * Name